### ISSUE TITLE
Immediate vs. Carryover Response percentage

### ISSUE THREAD

**Issue by satomi999** [facebookexperimental/Robyn#580]
## Describe issue
I came from the following post in the Robyn community !
https://www.facebook.com/groups/robynmmm/permalink/1346385012796295/

I have set the Weibull Adstock CDF parameters to c(0.0001) for some ads.
However, looking at the modeling output, there seems to be a carryover effect. (red ads)
![image](https://user-images.githubusercontent.com/98081674/208834966-bc014975-f0aa-46d2-a546-30b2b4081665.png)

I was understand with the answer on Robyn Community. (Thanks! @gufengzhou)

> Settings shape and scale to very small doesn't mean adstock will be 0. 

But It seems strange that there're a carryover effect over 40% even though the adstock is as close to zero as possible.
（By the way, I use daily data.）

I tried setting both shape and scale to 0, but I got an error.

```R
Error in check_hyper_limits(hyperparameters_ordered, "shapes") :
FBIG_S_shapes's hyperparameter must have lower bound >0
```

It would be nice if shape/scale could be set to 0 as well, but if there is a reason why it cannot be set to 0, I would appreciate it if you could tell me :)

Thanks!

## Environment & Robyn version
- Robyn version 3.9.0
- R version 4.2.1

**Comment by gufengzhou** [comment]
Can you please test the update and see if it works? You  can set shape and scale to 0 now to have 0 adstock / carryover for a channel.

**Comment by satomi999** [comment]
I was able to confirm the output when scale and shape were set to 0. (red ads)
Thank you for dealing with it so quickly!!

![image](https://user-images.githubusercontent.com/98081674/209043102-d13775ff-d448-4ea5-b723-797dc00e4828.png)

**Comment by jaylin1** [comment]
Robyn 3.10.3 is having issues with scales being equal to 0 again. I'm running an old model exactly as is on a newer version of Robyn and it is giving me an error. Setting the range to non-zero allows the model to run without issue. Is this an intentional change?
getting an error : Error in { : 
  task 1 failed - "'names' attribute [20] must be the same length as the vector [18]"
  
  @gufengzhou

**Comment by jstrome-lmp** [comment]
Hi there, this is still not resolved as of version 3.10.7.  Would anyone be able to take a look here?  Thank you for the great product!

**Comment by gufengzhou** [comment]
Hi @jstrome-lmp and @jaylin1, thanks for reporting. this is fixed is the latest v3.10.7.9000.
